community/concepts/cred.md

```diff
@@ -0,0 +1,20 @@
+# Cred
+
+SourceCred, as the name might suggest, is all about attributing cred. Cred is a metric that describes every contribution and contributor in a project to give a sense of how important they were.
+
+For example, forum posts can earn cred. If I post on the [sourcecred discourse](https://discourse.sourcecred.io/) and earn 5 cred, and another user posts and their post earns 10 cred, we can say that the other post is considered twice as important as my post.
+
+The contributions are arranged in a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)) where contributions are nodes, and have edges indicating how they relate to other contributions. For example, if someone writes a reply to my post, there will be an edge connecting the reply and my post. Edges have "types" that give some information about what kind of edge they are; that edge might have an "IS_REPLY" type. The connections are also directional, meaning their are relationships that connect nodes to it, and relationships that connect nodes from it. An edge without a direction is considered bi-directional.
```

Comment:
>For example, if someone writes a reply to my post, there will be an edge connecting the reply and my post. Edges have "types" that give some information about what kind of edge they are; that edge might have an "IS_REPLY" type.

The `;` breaking up the last sentence fragment is a little hard for me to parse. Suggested rewording for flow:

>For example, if someone writes a reply to my post, there will be an edge connecting the reply and my post. Edges have "types" that give some information about what kind of edge they are. So the reply to my post might have the edge type "IS_REPLY".

Reply:
Fixed locally.